src/components/tooltip/tooltip.js

```diff
@@ -10,6 +37,123 @@ import {h, Component} from 'preact';
  * @extends {Component}
  */
 class Tooltip extends Component {
+  state: Object;
+  _hoverTimeout: ?number;
+  textElement: HTMLSpanElement;
+  lastAlternativeTypeIndex: number = -1;
+
+  /**
+   * default component props
+   * @type {Object}
+   * @memberof Tooltip
+   */
+  static defaultProps = {
+    type: ToolTipType.Top,
+    maxWidth: '240px'
+  };
+
+  /**
+   * @constructor
+   * @param {*} props props
+   * @memberof Tooltip
+   */
+  constructor(props: Object) {
+    super(props);
```

Comment:
no need for constructor if there's no implementation in it

Reply:
removed